pkgs/tools/security/rainbowcrack/default.nix

```
@@ -0,0 +1,33 @@
+let
+  pkgs = import <nixpkgs> {  };
+
+in
+pkgs.stdenv.mkDerivation rec {
+  pname = "rainbowcrack";
+  version = "1.8-0kali1";
+
+  src = pkgs.fetchFromGitLab {
+    group = "kalilinux";
+    owner = "packages";
+    repo = "rainbowcrack";
+    rev = "kali/${version}";
+    hash = "sha256-y6jdFVQSO80YLLOitPAY8TVMUsi58B6+3bFoncwr+Qo=";
+  };
+
+  passthru = {
+      NIXPKGS_ALLOW_UNFREE = "1";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r * $out/bin
+  '';
+
+  meta = with pkgs.lib; {
+    description = "Rainbowcrack packaging for Kali Linux";
```

This doesn't really describe what the package is very well. How about this?

```suggestion
    description = "Rainbow table generator to be used for password cracking";
```